test/testdata/expected/TestArray.go

```diff
@@ -17,8 +17,8 @@ func (ta TestArray) Structs() []EmptyStruct {
 }`
 
 var TestFixedSizedArrayGetter string = `
-func (ta TestArray) FixedSizedArray() []uint32 {
```

Comment:
?

PS: They just enabled the new GitHub review feature for me/group/repository. I will use that to make the review.